### PR TITLE
[Independent-Publisher] Added ChuckNorris.IO connector

### PR DESCRIPTION
**Pull request by Laskewitz**
---
When submitting a connector, please check the following conditions for your PR.

- [x] I attest that the connector works and I verfied by deploying and testing all the operations.
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`.

### CODE REVIEW

File: independent-publisher-connectors/ChuckNorris.io/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "icon_url": {
+                  "type": "string",
+                  "description": "icon_url",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/ChuckNorris.io/apiDefinition.swagger.json:43]: @Laskewitz Can you please update it to `Icon URL`

File: independent-publisher-connectors/ChuckNorris.io/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "total": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "total",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/ChuckNorris.io/apiDefinition.swagger.json:118]: @Laskewitz Can you please update it to `Total`

File: independent-publisher-connectors/ChuckNorris.io/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+                    "type": "object",
+                    "properties": {
+                      "categories": {
+                        "type": "array",
+                        "items": {},
+                        "description": "categories"
+                      },
+                      "created_at": {
+                        "type": "string",
+                        "description": "created_at",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/ChuckNorris.io/apiDefinition.swagger.json:134]: @Laskewitz Can you please update it to `Created at`.

Documentation - [Description values of properties](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#operation-and-parameter-summaries-and-descriptions)

Applies to all the similar description values in the swagger.

File: independent-publisher-connectors/ChuckNorris.io/apiProperties.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#000000",
+    "capabilities": [],
+    "publisher": "Daniel Laskewitz"
```
Comment (sriyen-msft) [inline independent-publisher-connectors/ChuckNorris.io/apiProperties.json:6]: @Laskewitz Can you please add `stackOwner` as well

Documentation - [Adding metadata](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#adding-metadata)

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Review by Laskewitz — COMMENTED** [review]
@sriyen-msft I did all the changes!

**Comment by mamurshe** [comment]
[[certify-connector]]

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
Deployment pending..

File: independent-publisher-connectors/ChuckNorris.io/README.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+# Chuck Norris (ChuckNorris.IO)
+
+Chuck Norris facts are satirical factoids about martial artist and actor Chuck Norris that have become an Internet phenomenon and as a result have become widespread in popular culture. The 'facts' are normally absurd hyperbolic claims about Norris' toughness, attitude, virility, sophistication, and masculinity.
+
+Chuck Norris facts have spread around the world, leading not only to translated versions, but also spawning localized versions mentioning country-specific advertisements and other Internet phenomena. Allusions are also sometimes made to his use of roundhouse kicks to perform seemingly any task, his large amount of body hair with specific regard to his beard, and his role in the action television series Walker, Texas Ranger.
+
+[chucknorris.io](https://chucknorris.io) is a free JSON API for hand curated Chuck Norris facts.
+
+## Prerequisites
```
Comment (sriyen-msft) [inline independent-publisher-connectors/ChuckNorris.io/README.md:9]: @Laskewitz Please find a [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/readme.md) for readme.md file and add the required sections.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/ChuckNorris.io/README.md (hunk 5 above)
Comment (Laskewitz) [reply independent-publisher-connectors/ChuckNorris.io/README.md:11]: @sriyen-msft readme updated.

**Comment by sriyen-msft** [comment]
@Laskewitz Update - We are preparing your connector for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes between 7-10 business days. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps. Thanks

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by sriyen-msft** [comment]
@Laskewitz  Deployment freeze notice: Due to Thanksgiving week in the United States, we have temporarily halted the deployments of all the connectors till Tuesday, 30th November 2021.

We will begin the deployments on Wednesday, 1st December 2021. If your connector is queued up for Wednesday (1st December 2021) deployment schedule, the production deployment of your connector in all regions will be expected to be completed by Thursday, 9th December 2021. Thanks

**Comment by sriyen-msft** [comment]
@Laskewitz Apologies for the delay in response. 

Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program. Thanks

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**